server-provider/src/main/java/org/sonatype/tests/http/server/jetty/impl/JettyServerProvider.java

```diff
@@ -238,11 +241,11 @@ public void addUser(String user, Object password) {
         addCertificate(user, (CertificateHolder) password);
       }
       catch (Exception e) {
-        throw Throwables.propagate(e);
```

Comment:
`Throwables.propagate` is deprecated in recent Guava versions, removed usage to simplify future upgrades.